### ISSUE TITLE
Syntax not work with Marshmallow 3

### ISSUE THREAD

**Issue by zxyviopond** [lyft/toasted-marshmallow#14]
not compatible with Marshmallow 3 syntax.

**Comment by christian-storm** [comment]
I'm anxious to use this with the Marshmallow 3 as well.  Any pointers on how best to do this?

**Comment by pwaldo2** [comment]
Agreed, I'm booting up a test project and noticed that I need to do xyz_schema.data.field_name instead of xyz_schema.field_name. I also had to completely reinstall marshmallow, flask_marshmallow and marshmallow-sqlalchemy after uninstalling toastedmarshmallow.

**Comment by four43** [comment]
@rowillia are you guys (lyft folks) motivated to do a v3 anytime soon or is that out a ways for you?

EDIT (after playing with marshmallow for a few days): It seems that v3 is plenty performant stock for my needs. I'm curious at a scale like Lyft what that looks like though.
